prisma/seed.ts

```diff
@@ -19,15 +19,15 @@ function createTextStage() {
     type: StageType.TEXT,
     title: faker.random.words(5),
     text,
-    turns: hasTurns ? faker.datatype.number(12) : undefined,
+    turns: hasTurns ? faker.datatype.number({ min: 1, max: 12 }) : undefined,
   };
 }
 
 function createPollStage() {
   return {
     type: StageType.POLL,
     title: faker.random.words(5),
-    winner_points: faker.datatype.boolean() ? faker.datatype.number(1000) : null,
+    winner_points: faker.datatype.boolean() ? faker.datatype.number({ min: 100, max: 1000, precision: 100 }) : null,
```

Comment:
I'm not sure if precision does what you think it does here, see https://fakerjs.dev/api/datatype.html#number